### ISSUE TITLE
Commander

### ISSUE THREAD

**Issue by Jawbone999** [siggame/MegaMinerAI-Game-Ideas#74]
# TL;DR
- PvP king of the hill game
- Diagonally symmetrical map of islands with large central island
- Players start in opposite corner islands with a single base
- Use boats, planes, tanks, and caravans which each have different jobs/capabilities

# Story
In this game, two civilizations are at war: war over who gets to own the most incredible tropical island in the area! According to squatter's rights, if you live somewhere long enough, the place legally becomes yours - so deploy your troops and set up base on the island!

# Gameplay

**Victory**
Victory points are gained by having more units in the central capture zone than your opponent at the end of the opponent's turn. Being able to score points only after the opponent's turn helps to eliminate any turn-order bias. If both player's tie at the end of the game, then victory is determined by the number of bases built. If that's a tie, then it's number of units owned. If that's also a tie (jeez) then it's amount of money.

**Map**
The 60x40 or so map should feature:
- Diagonal symmetry
- Corner islands where the player's start with a single base
- Random small islands
- A central capture zone which consists of a large island and some water (so both tanks and boats can capture it)
- Random water tiles have ocean heat vents which are used to gain energy

Below is a depiction of how a map might look:
![CommandMap](https://user-images.githubusercontent.com/26576973/99564360-f4429b00-298f-11eb-8545-7371a67768d7.png)

And here's the same map, but with the capture zone shown:
![CommandMapOverlay](https://user-images.githubusercontent.com/26576973/99564359-f3aa0480-298f-11eb-9ba6-7b6cd3a1417a.png)

**Units**
Each player can spawn units on their bases. They are as follows:
| Unit | Description |
|---|---|
| **Tank** | Unit that can move on land. Capable of converting sea tiles into land to allow itself to move between islands. Can attack all units, but best against boats. |
| **Boat** | Unit that can move on water. Must be built on a coastal base tile. Can capture heat vents. Can attack all units, but best against planes. |
| **Plane** | Flying unit that must land on an allied base after flying for a while. Can destroy land by bombing it. Can attack all units, but best against tanks. Cannot be used to capture the central island zone. |
| **Caravan** | Slow land unit that is used to build bases. Cannot capture the zone or even build in it. Destroyed when used to build a base. |

All units can deposit themselves into a base's storage. This allows them to slowly heal, but they cannot be taken out until next turn.

# Summary
This is meant to be a Civilization-style war game. Players are expected to figure out best positions for new bases and bridges, and the ideal attack strategies. There may be a need for a money resource that can be gathered by tanks, since all resources come from the sea.

**Comment by EvGym** [comment]
you could have river tiles that support both boats and units to allow further use of both types

**Comment by JeffreyStrahm** [comment]
You can have river tiles that support both boats and land units. This should help with movement and usability. May end up in only boats being used though

**Comment by Jawbone999** [comment]
That's a good idea! One other suggestion was that the islands could be bigger so that any bridges would be only a few tiles long.
